### ISSUE TITLE
Color contrast for cells and labels violates WCAG 2.0 AA requirements

### ISSUE THREAD

**Issue by OwenEdwards** [kensgists/apt#1]
Most of the **Support Levels** labels, and one of the **Priority Levels** labels, fail WCAG 2.0 AA requirements. The text is 16px (12 point) bold (for Support Levels), or normal (for Priority Levels).

Standard text less than 18 point (or less than 14 point if bold) must have a luminosity contrast ratio of 4.5:1 or more.

Text 18 point or larger (14 point or larger if bold) must have a luminosity contrast ratio of 3:1 or more.

Several resources are available to test luminosity contrast ratio, and to select foreground/background color pairs which comply with WCAG 2.0 AA requirements.
